frontend/src/features/queue/components/ApplicationInQueueReview.tsx

```diff
@@ -58,7 +54,7 @@ export const ApplicationInQueueReview = ({
   const [hasAttemptedSubmission, setHasAttemptedSubmission] = useState(false);
 
   const handleEdit = () => {
-    navigate(APPLICATION_QUEUE_ROUTES.EDIT(companyId, applicationData?.permitId));
+    return;
```

Comment:
I think you also need to consider either removing the edit button completely for the time being (since it's not being used), or make it so that the button is temporarily disabled/unavailable (so that the user wouldn't be confused clicking on the button and nothing's happening)